core/services/relay/evm/mercury/transmitter.go

```diff
@@ -380,6 +381,25 @@ func (mt *mercuryTransmitter) HealthReport() map[string]error {
 	return report
 }
 
+func (mt *mercuryTransmitter) sendToTrigger(report ocrtypes.Report, rs [][32]byte, ss [][32]byte, vs [32]byte) error {
+	var rsUnsized [][]byte
+	var ssUnsized [][]byte
+	for idx := range rs {
+		rsUnsized = append(rsUnsized, rs[idx][:])
+		ssUnsized = append(ssUnsized, ss[idx][:])
+	}
+	converted := capMercury.FeedReport{
+		FeedID:     mt.feedID.Hex(),
+		FullReport: report,
+		Rs:         rsUnsized,
+		Ss:         ssUnsized,
+		Vs:         vs[:],
+		// NOTE: Skipping fields derived from FullReport, they will be filled out at a later stage
+		// after decoding and validating signatures.
+	}
+	return mt.triggerCapability.ProcessReport([]capMercury.FeedReport{converted})
```

Comment:
We are assuming this is async/non-blocking?

Reply:
Yes, non-blocking in a networking sense. It only saves the report in a local map under lock - implementation here: https://github.com/smartcontractkit/chainlink-common/blob/main/pkg/capabilities/triggers/mercury_trigger.go#L82